NESTOR_costmap:
  global_frame: /map
  robot_base_frame: /base_link
  transform_tolerance: 0.05
  update_frequency: 1.0
  publish_frequency: 0.0
  max_obstacle_height: 2.0
  obstacle_range: 3.5
  raytrace_range: 4.0
  cost_scaling_factor: 10.0
  #footprint: [[-0.1, -0.1], [-0.1, 0.1], [0.1, 0.1], [0.1, -0.1]]
  robot_radius: 0.3
  inflation_radius: 0.31
  observation_sources: hokuyo_scan_sensor neato_scan_sensor
  hokuyo_scan_sensor: {sensor_frame: /laser, data_type: LaserScan, topic: /scan, expected_update_rate: 5, marking: true, clearing: true}
  neato_scan_sensor: {sensor_frame: /neato_laser, data_type: LaserScan, topic: /scan, expected_update_rate: 2, marking: true, clearing: true}
  static_map: true
  resolution: 0.05
  unknown_cost_value: 50
  map_topic: map
  map_type: costmap
  track_unknown_space: true

NESTOR_planner:
  allow_unknown: true
  visualize_potential: true

